.changeset/brave-ducks-relax.md

```diff
@@ -0,0 +1,6 @@
+---
+'@firebase/auth': minor
```

Comment:
Not sure if the changeset is supposed to be included in this PR or only when merging to mainline?

Reply:
Yeah only do the changeset when merging back into `master` branch. You can ignore the CI check on these sub-PRs

Reply:
Ohh got it, I think this is going to be the only PR for this feature, so I'll just leave this here for now